sql/mob_pools.sql

```diff
@@ -2720,7 +2720,7 @@ INSERT INTO `mob_pools` VALUES (2659,'Mikiruru','Mikiruru',196,0x000014070000000
 INSERT INTO `mob_pools` VALUES (2660,'Mildaunegeux','Mildaunegeux',359,0x0500D90100000000000000000000000000000000,1,1,1,180,100,0,1,1,1,18,513,32,231,157,0,0,0,1,0,359);
 INSERT INTO `mob_pools` VALUES (2661,'Millenary_Mossback','Millenary_Mossback',2,0x0000900100000000000000000000000000000000,1,1,7,380,100,0,1,1,1,0,0,0,7,133,0,0,0,1,0,2);
 INSERT INTO `mob_pools` VALUES (2662,'Million_Eyes','Million_Eyes',139,0x0000800100000000000000000000000000000000,4,4,12,280,100,0,1,0,0,0,0,0,151,133,0,0,10,1,0,139);
-INSERT INTO `mob_pools` VALUES (2663,'Mimas','Mimas',126,0x0000D00200000000000000000000000000000000,1,2,6,240,100,0,1,0,1,2,0,32,178,159,0,0,0,1,0,126);
+INSERT INTO `mob_pools` VALUES (2663,'Mimas','Mimas',126,0x0000D00200000000000000000000000000000000,2,2,6,240,100,0,1,0,1,2,0,32,178,159,0,0,0,1,0,126);
```

Comment:
Technically this might not be wrong. He single-swings and animates as one attack per round even though his 2hr is hundred fists. Wiki often mistakes 2hr for equaling job but a lot of the time that isn't the case.

The other possibility is that his main job really is monk and something else should force him to single-swing. We _could_ theoretically calculate his stats from retail damage samples to find out for sure.

Reply:
..I'll leave this for someone else to make the call on, coz I dunno for sure the right answer.

Reply:
i see, i think you are right, makes more sense him being war with mnk 2hr, than being a mnk with single attack

just remove it?

Reply:
can, we can figure out for sure which it should be later if you'd rather leave it be for now

Reply:
did so